package.json

```diff
@@ -17,6 +17,10 @@
     "next": "13.4.4",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "sharp": "^0.32.1",
```

Comment:
Why do you need `sharp`

Reply:
Yea, but just installing it doesn't do the trick :D
Pls remove it if you are not using ig